aktuelt/handlers.py

```diff
@@ -0,0 +1,28 @@
+# Extends https://github.com/wagtail/wagtail/blob/main/wagtail/images/rich_text/__init__.py#L8
+from django.utils.html import escape
+from wagtail.images.formats import get_image_format
+from wagtail.images.rich_text import ImageEmbedHandler
+
+
+class CustomImageEmbedHandler(ImageEmbedHandler):
```

Comment:
A handler is how Wagtail converts internal xml/html-like format into frontend html. The current example works but should ideally be expanded to include srcset of all relevant sizes based on output of https://github.com/gathering/tgno-backend/blob/main/aktuelt/serializers.py#L11 (which is used for other news article images)